config.yaml

```diff
@@ -1,3 +1,5 @@
+settings:
```

Comment:
I missed this file earlier. I think toml is a better configuration language for an application like this. I think yaml is fine for now, but if we work on this further I think we should convert this to toml.